conformance/lang/expressions/range-expr/range_expr_exclusive.balt

```diff
@@ -0,0 +1,5877 @@
+Test-Case: output
+Description: Test range expression (exclusive) when values of both expressions are positive integers.
+Labels: range-expr, iterable-exclusive, array-type, list-constructor-expr, member-access-expr, int, DecimalNumber,
+        additive-expr
+
+function init() {
+    int startIndex = 0;
+    int endIndex = 6;
+    int count = 0;
+    int[] arr = [];
+    foreach int i in startIndex ..< endIndex {
+          arr[count] = i;  
```

Comment:
The label for this should probably be member-access-lvexpr. But shall we use array:push instead?